src/documents/IoCTestingFramework.md

```diff
@@ -122,151 +116,398 @@ fluid.tests.globalCatTest = function (catt) {
+            <td><code>{String}</code></td>
+            <td>If present, will override the key of this member of <code>elements</code> in representing the namespace of this element.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+The members `gradeNames` and `options` in the above table collectively designate an Infusion component derived from `fluid.test.sequenceElement` &mdash; this component will be 
+instantiated as a child component of the `fluid.test.testCaseHolder` and its options member `sequence` will be evaluated. This member `sequence` holds one or more 
+[`fixture elements`](#supported-fixture-records) just as seen in the top-level [`sequence`](#-modules-tests-and-sequence-) element.
+
+Any fixture elements listed at the traditional top-level `sequence` member will be grandfathered in to this system with a namespace of `sequence` &mdash; for example, a `testSequenceElement` with
+a priority of `"before:sequence"` will be sorted before these elements.
+
+#### Example of sequence building using `sequenceGrade`
+
+Here is a simple example of building up a sequence of elements piece by piece, using [priority](Priorities.md) constraints of the type `after:<namespace>` and `before:<namespace>`. At the top
+level, we define a compound `testEnvironment` and `testCaseHolder` that defines a single module holding a single test, referencing the grade `fluid.tests.elementPrioritySequence` as 
+designating the overall sequence to be built up as a `sequenceGrade`. This grade is defined with four `testSequenceElement` entries, whose namespaces are `check`, `end`, `postBeginning` and `beginning`
+(deliberately defined in a different order to that which they will be executed in). These elements each specify a `priority` element to guide the order in which they should be sorted into a sequence.
+If you run this example, you will see them executed in the order `beginning`, `postBeginning`, `sequence` (the grandfathered-in top-level sequence), `end` and finally `check`.
+
+The sequence makes use of two reusable `sequenceElement` grades, `fluid.tests.elementPriority.log` which defines a sequence element which logs a console message (this fixture element uses
+the [compact syntax](Invokers.md#compact-format-for-invokers) for encoding simple argument lists together with the function), and `fluid.tests.elementPriority.check` which defines
+a sequence element making a jqUnit assertion which always passes.
+
+```javascript
+fluid.defaults("fluid.tests.elementPriority.log", {
+    gradeNames: "fluid.test.sequenceElement",
+    sequence: [{
+        func: "fluid.log({that}.options.message)"
+    }]
+});
+
+fluid.defaults("fluid.tests.elementPriority.check", {
+    gradeNames: "fluid.test.sequenceElement",
+    sequence: [{
+        func: "jqUnit.assert",
+        args: "I am the check, right at the end"
+    }]
+});
+
+fluid.defaults("fluid.tests.elementPrioritySequence", {
+    gradeNames: "fluid.test.sequence",
+    sequenceElements: {
```

Comment:
I just confirmed by copying and pasting the examples from the docs, if you use `sequenceElements`, you end up with cryptic messages like:

```
16:13:29.838:  jq: FAIL: Module "Priority-driven grade budding" Test name "Simple sequence of 4 active elements" - Message: Died on test #1     at Object.asyncTest (/Users/duhrer/Source/rtf/ul-image-api/node_modules/infusion/tests/lib/qunit/js/qunit.js:405:9): Cannot set property 'sequence' of undefined
16:13:29.838:  jq: Source: TypeError: Cannot set property 'sequence' of undefined
    at Object.fluid.test.resolveSequence (/Users/duhrer/Source/rtf/ul-image-api/node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js:498:31)
    at Object.fluid.test.sequenceExecutor (/Users/duhrer/Source/rtf/ul-image-api/node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js:509:49)
    at Object.testFunc (/Users/duhrer/Source/rtf/ul-image-api/node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js:630:32)
16:13:29.856:  jq: Test concluded - Module "Priority-driven grade budding" Test name "Simple sequence of 4 active elements": 0/1 passed - FAIL
```

Updating the examples to use `elements` immediately and totally fixes the problem.